### DISCUSSION TITLE
Themed images in .md files (dark mode/light mode)

### DISCUSSION THREAD

**Discussion by BieleckiLtd** [community/community#41758] [Mobile] [answered]
### Select Topic Area

Question

### Body

What is the correct syntax to embed themed images in markdown files? [This](https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/) blogpost suggest to use `#gh-dark-mode-only` or `#gh-light-mode-only` appendix and this works in mobile, however [this doc](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) says not to use it as it is deprecated and use `<picture></picture>` syntax instead. When I do so &#65533; is displayed in mobile client:

![0AA4BD4C-E652-4BE8-9F3C-E6B0A5356DA6](https://user-images.githubusercontent.com/57046415/207691309-dd02d0b9-9687-47bf-bd4b-5a31c5d6238e.jpeg)

**Comment by aconsuegra** [comment, marked as answer]
Hi @BieleckiLtd, thanks for reaching out and for your feedback! Unfortunately, the Mobile apps still don't support `<picture>` for dark/light mode images, and we don't know exactly when we'll be able to add support for it. I have created an issue on our end to track your request.

**Reply by tenthirtyam** [reply]
Any update on this issue? I'm still seeing this on mobile.

**Reply by aconsuegra** [reply]
Hi @tenthirtyam. Sadly, this is still an issue on the Mobile apps, and we don't have an ETA for a fix.

**Reply by simonprovost** [reply]
any news?
